src/keyboard/mod.rs

```diff
@@ -356,6 +352,22 @@ impl Drop for KbState {
     }
 }
 
+/// Determines the behaviour of key repetition
+#[derive(PartialEq)]
+pub enum KeyRepeatKind {
+    /// keys will not be repeated
+    None,
```

Comment:
Given we have split the repeat handling into a new function, this variant is not necessary any more.